app/main.py

```diff
@@ -3,15 +3,7 @@
 
 
 def battle(knights_config: dict) -> dict:
-    for knight, info in knights_config.items():
-        knight = Knight(name=info["name"],
-                        power=info["power"],
-                        hp=info["hp"])
-        Knight.knights_dict[info["name"]] = knight
-        knight_update = Knight.knights_dict[info["name"]]
-        knight_update.update(armour=info["armour"],
-                             weapon=info["weapon"],
-                             potion=info["potion"])
+    Knight.upgrade(knights_config=knights_config)
```

Comment:
no need to kall this method here, u can just call self.update in init method

Reply:
> no need to kall this method here, u can just call self.update in init method
@YuriiKindrat 
I don't understand what needs to be done, I was at the Q&A and the mentor told me that he also doesn't understand what needs to be done, so can you explain

Reply:
@MaybeTI u have 2 similar names for methods, so i was confused